### PR TITLE
add missing include in RefHolder_.h needed for gcc10/modules 

### PR DESCRIPTION
**Pull request by davidlange6**
Fixes compilation error in Friday's modules IB.

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by makortel** [comment]
2026D60 28234.0 all_mini_OldVSNew shows a small difference for patMETs_slimmedMETsNoHF for which I opened an issue https://github.com/cms-sw/cmssw/issues/36552

**Comment by makortel** [comment]
@cmsbuild, please test

**Comment by makortel** [comment]
+1

The difference in  patMETs_slimmedMETsNoHF is still visible, but it appears in other PR tests too (#36552) so unlikely to be caused by this PR.

**Comment by qliphy** [comment]
+1

File: DataFormats/Common/interface/RefHolder_.h [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include "FWCore/Utilities/interface/OffsetToBase.h"
 #include <memory>
 #include <typeinfo>
+#include <string>
```
Comment (Dr15Jones) [inline DataFormats/Common/interface/RefHolder_.h:10]: Why was this needed? `std::string` is not used anywhere in this file. What was the missing symbol needed?

Comment (Dr15Jones) [reply DataFormats/Common/interface/RefHolder_.h:10]: Never mind. I see `std::string&`. I shouldn't write comments when I haven't finished my coffee :(.
